javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/matcher/GlobalIgnoresMatcher.java

```diff
@@ -57,8 +57,7 @@ public boolean matches(T target) {
       return false;
     }
 
-    if (name.startsWith("org.gradle.")
```

Comment:
This seems like a reasonable addition even if it doesn't solve our deadlock. If nothing else, it could speed up the build a bit by having less to run instrumentation on?